Testing/SystemTests/tests/framework/D7YIGPositionCalibrationTest.py

```diff
@@ -0,0 +1,137 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+import systemtesting
+from mantid.simpleapi import *
+from mantid.api import ITableWorkspace
+
+import math
+import os.path
+from os import path
+import xml.etree.ElementTree as ET
+
+
+class D7YIGPositionCalibrationTest(systemtesting.MantidSystemTest):
```

Comment:
Not critical, but you could as well have three test classes, one for each wavelength.